php-nginx/php.ini

```diff
@@ -478,7 +478,7 @@ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
 ; Development Value: On
 ; Production Value: Off
 ; http://php.net/display-errors
-display_errors = Off
+display_errors = stderr
```

Comment:
I'm puzzled. In my experiment, I don't see anything logged in a file specified by `error_log` in `php.ini`.